main.py

```diff
@@ -59,6 +58,31 @@ def on_keypress(bit_index):
   clock.tick(60)
   pygame.display.flip()
 
+  if not game.state_of_game:
```

Comment:
For one thing, I had to check that this was in fact a boolean. But also, state is a synonym for condition, its the way that something is or how it is arranged. Like the state of matter is usually one of solid, liquid or gas. But we need to know what the state is. `state_of_game` sounds like it will contain the state, but `True` and `False` to refer to the state fo a game as a whole makes little sense.